pkg/asset/installconfig/azure/validation.go

```diff
@@ -189,7 +194,10 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 			ultraSSDCapability = defaultUltraSSDCapability
 		}
 		ultraSSDEnabled := strings.EqualFold(ultraSSDCapability, "Enabled")
-		allErrs = append(allErrs, ValidateInstanceType(client, field.NewPath("controlPlane", "platform", "azure"), ic.Azure.Region, instanceType, diskType, controlPlaneReq, ultraSSDEnabled)...)
+		if acceleratedNetworking == "" {
+			acceleratedNetworking = string(aztypes.AcceleratedNetworkingAccelerated)
+		}
```

Comment:
If the instance type does not support accelerated networking, should we use basic networking when the user does not make an explicit choice?

Reply:
Yes, this is probably the best experience.

Reply:
We are going to leave this for now based upon our slack discussion.